pkg/util/log/ambient_context.go

```diff
@@ -158,7 +158,7 @@ func (ac *AmbientContext) annotateCtxInternal(ctx context.Context) context.Conte
 //
 // The caller is responsible for closing the span (via Span.Finish).
 func (ac *AmbientContext) AnnotateCtxWithSpan(
-	ctx context.Context, opName string,
+	ctx context.Context, opName string, os ...tracing.SpanOption,
```

Comment:
The diffs in these files can be reverted.

Reply:
done.